app/controllers/bookings_controller.rb

```diff
@@ -5,5 +5,21 @@ def new
   end
 
   def create
+    @instrument = Instrument.find(params[:instrument_id])
+    @booking = Booking.new(booking_params)
+    @booking.user = current_user
```

Comment:
You could just add:
`before_action :authenticate_user!` in the first line of the controller, then all controller actions here need an authenticated user

Reply:
Exactly, we added the authentication the other pull request